.github/workflows/cd.yml

```diff
@@ -59,7 +59,6 @@ jobs:
           docker load -i docker-image/editor-article-store.tar
           make start_services
           IMAGE_TAG=${IMAGE_TAG} docker-compose up -d editor-article-store
-          docker exec localstack awslocal s3 cp /resources/articles/elife-54296-vor-r1.zip s3://kryia/
```

Comment:
Was taking this out intentional?

Reply:
Yep, We no longer need to pre-seed the DB to run functional tests